stdlib/public/Platform/Platform.swift

```diff
@@ -187,6 +209,7 @@ public func open(
   return _swift_Platform_open(path, oflag, 0)
 }
 
+#if !os(Windows) || CYGWIN
 public func open(
   _ path: UnsafePointer<CChar>,
   _ oflag: Int32,
```

Comment:
Because theres no `openat` equivalent atm.  This is possible to do and unhide this, but, Im trying to get an immediate port going.  It would be possible to get an implementation of `openat` and then unhide this in a follow up.